packages/emotion-theming/types/index.d.ts

```diff
@@ -21,6 +21,14 @@ export interface EmotionThemingModule<Theme> {
     withTheme<Props>(component: ComponentClass<Props> | SFC<Props>): ComponentClass<OptionalThemeProps<Props, Theme>>;
 }
 
-export const channel: string;
+export const channel: "__EMOTION_THEMING__";
 
-export function createBroadcast(initialState: any): any;
+export type BroadcastListener<Theme = {}> = (state: Theme) => void;
+
+export interface Broadcast<Theme = {}> {
+    publish: (nextState: Theme) => void;
+    subscribe: (listener: BroadcastListener<Theme>) => number;
+    unsubscribe: (unsubID: number) => void;
```

Comment:
Is there any reason for using arrow function type instead of method types?
(In TS, they are different in terms of assignability)

Reply:
No special reason. I guess the method type is better in that case?